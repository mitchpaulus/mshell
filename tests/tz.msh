2025-05-29T16:00 utcToCst str wl
2025-01-02T16:00 utcToCst str wl
