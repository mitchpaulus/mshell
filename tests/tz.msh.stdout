2025-05-29T11:00:00
2025-01-02T10:00:00
